src/node.h

```diff
@@ -1462,7 +1462,6 @@ class NODE_EXTERN CallbackScope {
   CallbackScope(CallbackScope&&) = delete;
 
  private:
-  void* resource_storage_global_;
```

Comment:
This structure is a public API and changing the private fields will change  the size of the struct and cause ABI incompatibility.

This field was named as `reserved_`.

This should be a semver-major change.

Reply:
Calling it `void* reserved_;` should be fine without a comment tbh -- this is a pretty common pattern in C/C++ APIs for ABI forward compatibility.

Reply:
Updated